server/modules/surveyRdb/repository/dataView/read.js

```diff
@@ -196,6 +196,25 @@ export const fetchViewData = async (params, client = db) => {
     : client.map(select, queryParams, _dbTransformCallbackSelect({ viewDataNodeDef, nodeDefCols, editMode }))
 }
 
+export const _fetchViewData = async (params) => {
+  const { survey, columnNodeDefs = [], entityDefUuid } = params
+
+  const nodeDef = Survey.getNodeDefByUuid(entityDefUuid)(survey)
+  const nodeDefCols = columnNodeDefs // [] //Survey.getNodeDefsByUuids(Query.getAttributeDefUuids(query))(survey)
+
+  const viewDataNodeDef = new ViewDataNodeDef(survey, nodeDef)
+
+  const queryBuilder = new SqlSelectBuilder()
+  queryBuilder.select('*')
```

Comment:
the view contains attributes up to the root entity. only child defs must be exported.